webpage/static_dumps/dyscover.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Dyscover</title>
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.9.1.min.js"></script>
+<script>
+function showInput(){
+  var location = document.getElementById("q_id").value;
+  var result = "";
+  $("#display").html("Fetching results...");
+  $.getJSON("http://localhost:8000/api/search/", {q:location}, function(data, status){
+    restaurant_list = data["Restaurants"];
```

Comment:
these variables defined without `var` will go in the global scope and cause memory leaks